atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -1187,23 +1188,16 @@ private static LockAndTimestampServices createRawLeaderServices(
                 leaderConfig,
                 userAgent);
         LeaderElectionService leader = localPaxosServices.leaderElectionService();
-        LockService localLock = ServiceCreator.instrumentService(
-                metricsManager.getRegistry(),
-                AwaitingLeadershipProxy.newProxyInstance(LockService.class, lock::get, leader),
-                LockService.class);
+        LockService localLock = AwaitingLeadershipProxy.newProxyInstance(LockService.class, lock::get, leader);
 
         ManagedTimestampService managedTimestampProxy =
                 AwaitingLeadershipProxy.newProxyInstance(ManagedTimestampService.class, time::get, leader);
 
-        TimestampService localTime = ServiceCreator.instrumentService(
-                metricsManager.getRegistry(),
-                managedTimestampProxy,
-                TimestampService.class);
+        // These facades are necessary because of the semantics of the JAX-RS algorithm (in particular, accepting
+        // just the managed timestamp service will *not* work.
```

Comment:
nit: close bracket (that was opened on the line above)

Reply:
good spot!